docs/api/index.rst

```diff
@@ -249,9 +249,9 @@ Record Objects
 ==============
 
 Each row (or composite type value) returned by calls to ``fetch*`` methods
-is represented by an instance of the ``Record`` object.  ``Record`` objects
-are similar to instances of ``collections.namedtuple`` and allow addressing
-of values either by a numeric index or by a field name:
+is represented by an instance of the :class:`~asyncpg.Record` object.
+``Record`` objects are similar to instances of ``collections.namedtuple`` and
```

Comment:
Records are actually closer to dicts. They use `__getitem__`, have `.items()`, `.values()` and `.keys()` methods.